### PR TITLE
SAIC-113 Test load generator implementation

### PR DESCRIPTION
**Pull request by plomakin**
Added generator and config file to generate load on "src" node in 
devlab-environment.
Added description about usage and configuration.

### CODE REVIEW

File: devlab/tests/prerequisite_actions.py [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+from keystoneclient.v2_0 import client as keystone
+
+import config
+
+NOVA_CLIENT_VERSION = '1.1'
+GLANCE_CLIENT_VERSION = '1'
+NEUTRON_CLIENT_VERSION = '2.0'
+
+
+class Silverlining_prerequsites():
```
Comment (roman-verchikov) [inline devlab/tests/prerequisite_actions.py]: Let's change the name to something neutral ;)

**Comment by Wasapl** [comment]
Merged build finished. Test PASSed.  http://jenkins-cf.vm.mirantis.net:8080/job/CloudFerry_PR_checks/212/

**Comment by Wasapl** [comment]
Merged build finished. Test FAILed.   http://jenkins-cf.vm.mirantis.net:8080/job/CloudFerry_PR_checks/213/

**Comment by Wasapl** [comment]
Merged build finished. Test PASSed.  http://jenkins-cf.vm.mirantis.net:8080/job/CloudFerry_PR_checks/214/

**Comment by Wasapl** [comment]
Merged build finished. Test PASSed.  http://jenkins-cf.vm.mirantis.net:8080/job/CloudFerry_PR_checks/215/

**Comment by Wasapl** [comment]
Merged build finished. Test PASSed.  http://jenkins-cf.vm.mirantis.net:8080/job/CloudFerry_PR_checks/216/

File: devlab/tests/prerequisite_actions.py [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+from keystoneclient.v2_0 import client as keystone
+
+import config
+
+NOVA_CLIENT_VERSION = '1.1'
+GLANCE_CLIENT_VERSION = '1'
+NEUTRON_CLIENT_VERSION = '2.0'
+
+
+class Prerequsites():
```
Comment (roman-verchikov) [inline devlab/tests/prerequisite_actions.py]: Classname misspelt, should be "prerequisites"

**Comment by Wasapl** [comment]
Merged build finished. Test PASSed.  http://jenkins-cf.vm.mirantis.net:8080/job/CloudFerry_PR_checks/217/

**Comment by roman-verchikov** [comment]
+1

**Comment by miarmak** [comment]
Works for me. +1

**Comment by Wasapl** [comment]
Merged build finished. Test PASSed.  http://jenkins-cf.vm.mirantis.net:8080/job/CloudFerry_PR_checks/227/

**Comment by Wasapl** [comment]
Merged build finished. Test PASSed.  http://jenkins-cf.vm.mirantis.net:8080/job/CloudFerry_PR_checks/228/

File: devlab/tests/README.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
+Directory with tests for Cloudferry's devlab
+==========
+
+## Prerequsite action
```
Comment (roman-verchikov) [inline devlab/tests/README.md]: Typo: should be "prerequisite", not "Prerequsite"

**Comment by roman-verchikov** [comment]
Only a small type to fix, see inline

**Comment by plomakin** [comment]
Fixed

**Comment by Wasapl** [comment]
Merged build finished. Test PASSed.  http://jenkins-cf.vm.mirantis.net:8080/job/CloudFerry_PR_checks/237/

**Pull request merged**